lib/valueflow.cpp

```diff
@@ -7741,11 +7741,9 @@ struct ContainerExpressionAnalyzer : ExpressionAnalyzer {
         return tok->exprId() == expr->exprId() || (astIsIterator(tok) && isAliasOf(tok, expr->exprId()));
     }
 
-    Action isWritable(const Token* tok, Direction d) const override {
```

Comment:
In the future please comment these out instead (`Direction /*d*/`) so they are more visible. No need to change this PR if it goes through.

There might even be a clang-tidy checks for this.